jsonrpc/jsonrpc.go

```diff
@@ -4,14 +4,14 @@
 // Use []interface{} as the type of argument when sending and receiving methods.
 //
 // Positional arguments example:
-// 	server.Handle("add", func(client *rpc2.Client, args []interface{}, result *float64) error {
-// 		*result = args[0].(float64) + args[1].(float64)
-// 		return nil
-// 	})
 //
-//	var result float64
-// 	client.Call("add", []interface{}{1, 2}, &result)
+//	server.Handle("add", func(client *rpc2.Client, args []interface{}, result *float64) error {
+//		*result = args[0].(float64) + args[1].(float64)
+//		return nil
+//	})
 //
+//	var result float64
+//	client.Call("add", []interface{}{1, 2}, &result)
```

Comment:
I don't know why these lines seem changed. Can you fix that?

Reply:
Dude, how can you be so fast :D